Tests/ViewTests.swift

```diff
@@ -52,6 +52,13 @@ class ViewTests: QuickSpec {
                     expect(view.wasAnimateOnShowCalled).to(beTrue())
                 }
 
+                it("will disable user interaction in parent view for the time of animation") {
+                    view.show(accessibilityCompatible: false, onDismiss: {})
+                    expect(view.superview?.isUserInteractionEnabled).to(beFalse())
+                    expect(view.wasAnimationCompletionCalled).toEventually(beTrue())
+                    expect(view.superview?.isUserInteractionEnabled).to(beTrue())
```

Comment:
This only gets set after the animation completes so should this be `toEventually`?

Reply:
Yes but I expect the value to be true immediately after `wasAnimationCompletionCalled` is true

Reply:
Does the `toEventually` on `wasAnimationCompletionCalled` block until it evaluates true or times out? 
I don't think it does so it's curious that this is passing.

Reply:
You mean does it wait? Yes it does. `toEventually` is not async. If I removed `expect(view.wasAnimationCompletionCalled).toEventually(beTrue())` line then I'd have to change the expectation to `toEventually`

Reply:
OK - makes sense in that case.